.github/pull_request_template.md

```diff
@@ -14,7 +14,6 @@ Fixes # (issue)
 
 ## Checklist:
 
-- [ ] My code follows the style guidelines of this project
```

Comment:
While we auto-format the code with scalafmt, I think we may want to have a separate guidelines about "style" of code that is not related to formatting. That said, we don't have a style "guide" in this project, so I'm fine with removing this until we have one.